### DISCUSSION TITLE
uYouPlus default

### DISCUSSION THREAD

**Discussion by donnje** [qnblackcat/uYouPlus#862] [Q&A]
Hi,
Can uYouPlus be set by default when clicking a youtube link?

**Comment by yodaluca23** [comment]
> Hi,
> 
> Can uYouPlus be set by default when clicking a youtube link?

Check [here](https://github.com/qnblackcat/uYouPlus/wiki/FAQ#q-deep-link-aka-open-in-the-youtube-app-doesnt-work), and please read the FAQs before making a new discussion.

**Reply by abheizthe** [reply]
sorry but, how about this? https://github.com/CokePokes/YoutubeExtensions/issues/10

**Reply by yodaluca23** [reply]
> sorry but, how about this? https://github.com/CokePokes/YoutubeExtensions/issues/10

I actually had an even better idea of how we could do this I just have no idea how to code it… Lol because it allows you to use a different default, browser other than Safari and I think it would be working better and more reliable…

https://github.com/qnblackcat/uYouPlus/discussions/668
